src/components/PlayerProfile.vue

```diff
@@ -162,17 +163,18 @@ export default {
     $route: 'executeQuery',
   },
   methods: {
+    fromNow(date) {
+      return `${countdown(new Date(), date, countdown.ALL, 1)} ago`;
+    },
     formatDate(date) {
-      if (date == null) {
+      if (!date) {
         return '-';
       }
       const d = new Date(date);
-      return (d.getFullYear() === 1969) ? 'Never' : d.toLocaleString();
+      return d.toLocaleString();
     },
     getRanks() {
-      if (this.player === undefined || this.player.groups.length === 0) {
-        return 'None';
-      }
+      if (this.player?.groups?.length === 0) return 'None';
```

Comment:
This still throws for players without groups. You're checking explictly for `0`. I suggest to check if the value is falsy instead:
`if( ! this.player?.groups?.length )`